.devcontainer/devcontainer.json

```diff
@@ -8,7 +8,7 @@
       // Update the VARIANT arg to pick a version of Go: 1, 1.16, 1.17
       // Append -bullseye or -buster to pin to an OS version.
       // Use -bullseye variants on local arm64/Apple Silicon.
-      "VARIANT": "1.20-bullseye",
+      "VARIANT": "1-1.21-bullseye",
```

Comment:
Why is this `1-1.21` and not `1.21`?

Reply:
Because on generating this devcontainer config using devcontainers extension, by default it uses `1-1.21` instead of `1.21`.
NOTE: *I highly doubt it'll ever happen but if Go 1.21 gets a 2nd major release, `1.21` would use that & might cause an issue whereas `1-1.21` won't.*